src/Symfony/Component/Ldap/Tests/Security/CheckLdapCredentialsListenerTest.php

```diff
@@ -171,6 +196,9 @@ public static function queryForDnProvider(): iterable
         yield ['{user_identifier}', '{user_identifier}_test'];
     }
 
+    /**
+     * @group legacy
```

Comment:
why is this becoming a legacy tests ? Why would we remove this test when removing deprecated features related to this PR ?